app/models/waste_carriers_engine/past_registration.rb

```diff
@@ -31,7 +31,7 @@ def version_already_backed_up?(registration)
       return false if new_version_due_to_edit?
 
       # Collect all expires_on dates from past registrations
-      matching_expires_on = registration.past_registrations.map(&:expires_on)
+      matching_expires_on = registration.past_registrations.where(cause: :renewal).map(&:expires_on)
```

Comment:
What about any existing PastRegistrations which don't have this cause saved? I wonder if we will need to update them, although I think we can at least assume they are renewals.

Reply:
There won't be any new one I have made that param required. 
For past registrations (current, already in the data) with no cause, they won't affect the logic of this method. This is a guard clause that is almost never hit unless we run the RenewlCompletionService multiple times, and that only happen if we have a glitch/issue. Hence, I won't worry about that at all.

Reply:
I think I'm with @irisfaraway on this one. I'm not seeing how on day one of shipping this change that this wouldn't miss a `past_registration` with no cause and a matching expiry date.

Reply:
`miss a past_registration with no cause and a matching expiry date` 
I am saying It would, but is not important. Better to save an extra one (very very few chances of that, as we run this code only once) that to miss one. Happy to have a call whenever :)